bigchaindb/backend/localmongodb/query.py

```diff
@@ -77,3 +79,30 @@ def store_block(conn, block):
             .insert_one(block))
     except DuplicateKeyError:
         pass
+
+
+@register_query(LocalMongoDBConnection)
+def get_txids_filtered(conn, asset_id, operation=None):
+    match_create = {
+        'operation': 'CREATE',
```

Comment:
I just noticed we don't have any index on `operation` (this is true also for previous versions of BigchainDB), would it make sense to create one? If so, we can create an issue and solve it later.

Reply:
Good point #1893